### PR TITLE
Remove region from task definition

### LINKED ISSUE #541: Fix dag: national-day-after
Current dag isnt running. 

https://github.com/openclimatefix/ocf-infrastructure/blob/main/terraform/modules/services/airflow/dags/uk/dayafter-dag.py#L41

Need to remove `region` from task defintion

Similar to https://github.com/openclimatefix/ocf-infrastructure/pull/527, but this one was forgotten

Context
- We collect real time PVlive data, and day after PVLive data
- The national day after PVLive data is available about 1 hour before the rest of the GSP data
- Currently the later dag, to collect all GSPs, also collected National, as a backup

### PR DESCRIPTION
**Pull request by BreakingPitt**
# Pull Request

## Description

As requested in #541 remove the region from task definition in dayafter-dag.py

Fixes #541 

## Checklist:

- [X] My code follows [OCF's coding style guidelines](https://github.com/openclimatefix/.github/blob/main/coding_style.md)
- [X] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked my code and corrected any misspellings

### CODE REVIEW

File: terraform/modules/services/airflow/dags/uk/dayafter-dag.py [hunk 1]
```diff
@@ -25,10 +25,9 @@
 
 # Tasks can still be defined in terraform, or defined here
 
-region = 'uk'
 
 with DAG(
-    f'{region}-national-day-after',
```
Comment (peterdudfield) [inline terraform/modules/services/airflow/dags/uk/dayafter-dag.py:31]: this is fine to leave in

File: terraform/modules/services/airflow/dags/uk/dayafter-dag.py [hunk 2]
```diff
@@ -37,8 +36,8 @@
     dag1.doc_md = "Get National PVLive updated values"
 
     national_day_after = EcsRunTaskOperator(
-        task_id=f'{region}-national-day-after',
```
Comment (peterdudfield) [inline terraform/modules/services/airflow/dags/uk/dayafter-dag.py:40]: this is fine to leave int

File: terraform/modules/services/airflow/dags/uk/dayafter-dag.py [hunk 3]
```diff
@@ -37,8 +36,8 @@
...
-        task_definition=f'{region}-national-day-after',
+        task_id='national-day-after',
+        task_definition='national-day-after',
```
Comment (peterdudfield) [inline terraform/modules/services/airflow/dags/uk/dayafter-dag.py:40]: The change you made for`task_definition` is correct

File: terraform/modules/services/airflow/dags/uk/dayafter-dag.py (hunk 1 above)
Comment (BreakingPitt) [reply terraform/modules/services/airflow/dags/uk/dayafter-dag.py:31]: Ok!

**Comment by BreakingPitt** [comment]
@peterdudfield Sorry for the confusion, I misunderstood the objective of the change and thought I needed to remove the region throughout the entire file instead of just in the specific task definition. I've just corrected it and revert the unnecessary changes.

Can you review it to see if it is right? 

Best regards

File: terraform/modules/services/airflow/dags/uk/dayafter-dag.py [hunk 4]
```diff
@@ -55,7 +55,7 @@
     )
 
 with DAG(
-    f'{region}-gsp-day-after',
+    'gsp-day-after',
```
Comment (peterdudfield) [inline terraform/modules/services/airflow/dags/uk/dayafter-dag.py:58]: you can put this back in

File: terraform/modules/services/airflow/dags/uk/dayafter-dag.py [hunk 5]
```diff
@@ -65,7 +65,7 @@
     dag2.doc_md = "Get GSP PVLive updated values, and then triggers metrics DAG"
 
     gsp_day_after = EcsRunTaskOperator(
-        task_id=f'{region}-gsp-day-after',
```
Comment (peterdudfield) [inline terraform/modules/services/airflow/dags/uk/dayafter-dag.py:68]: you can put this back in

File: terraform/modules/services/airflow/dags/uk/dayafter-dag.py [hunk 6]
```diff
@@ -83,7 +83,7 @@
     )
 
     metrics = EcsRunTaskOperator(
-        task_id=f'{region}-metrics',
```
Comment (peterdudfield) [inline terraform/modules/services/airflow/dags/uk/dayafter-dag.py:86]: you can puit this back in

**Comment by peterdudfield** [comment]
Thanks @BreakingPitt Ive done this now in https://github.com/openclimatefix/ocf-infrastructure/pull/556, so Ill close this PR if thats ok. Thank you for your help

**Pull request closed without merging**